build.gradle

```diff
@@ -92,6 +92,31 @@ distributions {
 			}
 		}
 	}
+	latest {
+		contents {
```

Comment:
@aalmiray any way to refer to the main contents to avoid duplicating the same definition ?

Reply:
figured it out by using shared spec